pkg/kube/inject/testdata/inject/auth.non-default-service-account.yaml.injected

```diff
@@ -17,16 +17,14 @@ spec:
         prometheus.io/path: /stats/prometheus
         prometheus.io/port: "15020"
         prometheus.io/scrape: "true"
-        sidecar.istio.io/interceptionMode: REDIRECT
         sidecar.istio.io/status: '{"version":"","initContainers":["istio-init"],"containers":["istio-proxy"],"volumes":["istio-envoy","istio-data","istio-podinfo","istio-token","istiod-ca-cert"],"imagePullSecrets":null}'
-        traffic.sidecar.istio.io/excludeInboundPorts: "15020"
-        traffic.sidecar.istio.io/includeInboundPorts: "80"
-        traffic.sidecar.istio.io/includeOutboundIPRanges: '*'
       creationTimestamp: null
       labels:
         app: hello
         istio.io/rev: ""
         security.istio.io/tlsMode: istio
+        service.istio.io/canonical-name: hello
```

Comment:
You will find this diff on every file basically. *This is a bug fix*. The kube-inject label was adding a bunch of annotations it didn't need to, and missing the canonical labels